client/src/components/Modal/TextInputModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { View, Modal, TouchableOpacity, TouchableWithoutFeedback, Text, TextInput } from 'react-native';
+
+import styles from './styles';
+
+const TextInputModal = ({ closeModal, backModal, onChangeText, visible, onSave }) => (
+  <View>
+    <Modal
+      transparent
+      visible={visible}
+      onRequestClose={closeModal}
+      animationType="fade"
+    >
+      <TouchableWithoutFeedback
+        onPress={() => closeModal}
```

Comment:
you should just be able to use the following here. If it doesn't work make sure the method you're ultimately passing in as `closeModal` (from the user of `TextInputModal`) is properly bound.
```
onPress={closeModal}
```

Reply:
Sweet. fixed.